openmdao/docs/advanced_guide/derivs/partial_derivs_explicit.rst

```diff
@@ -24,48 +23,48 @@ Here is an example, based on the :ref:`Betz Limit Example <betz_limit_tutorial>`
+    There is support for when your derivatives are constant, and there is support for specifying derivatives in a sparse AIJ format.
+    The full details can be found in the :ref:`feature doc on specifying partials <feature_specify_partials>`.
 
-After you declare the non-zero partial derivatives, you need to implement the :code:`compute_partials` method to perform the actual
+After you declare the nonzero partial derivatives, you need to implement the :code:`compute_partials` method to perform the actual
 derivative computations.
 OpenMDAO will call this method whenever it needs to work with the partial derivatives.
 The values are stored in the Jacobian object, :code:`J`, and get used in the linear solutions that are necessary to compute model-level total derivatives.
 This API results in the assembly of a Jacobian matrix in memory.
 The :code:`compute_partials` API is the most appropriate way to declare derivatives in the vast majority of circumstances,
 and you should use it unless you have a good reason not to.
 
-Providing derivatives using the matrix-free API
-************************************************
+Providing Derivatives Using the Matrix-Free API
+***********************************************
 
 Sometimes you don't want to assemble the full partial-derivative Jacobian of your component in memory.
-We're not going to discuss why this might be the case in this tutorial.
-For now, let's just say that if matrix-assembly won't work for your application you are likely already well aware of this issue.
-If you can't imagine why you would want to use a matrix-free API, then don't worry about this.
-If you do need to work matrix-free, there is a :code:`compute_jacvec_product` API that you can use.
-See the :ref:`ExplicitComponent <comp-type-2-explicitcomp>` reference for an example of how to use it.
+The reasons why you might not want this are beyond the scope of this tutorial.
+For now, let's assume that if matrix assembly won't work for your application, that you are likely already well aware of this issue.
+So if you can't imagine why you would want to use a matrix-free API, you may disregard the following link.
+If you do need to work matrix-free, there is a :code:`compute_jacvec_product` API, examples of which can be found
+in the feature document for :ref:`ExplicitComponent <comp-type-2-explicitcomp>`.
 
 
-How Do I Know if My Derivatives Are Correct?
-**************************************************
+How Do I Know If My Derivatives Are Correct?
+********************************************
 
 It is really important, if you are going to provide analytic derivatives, that you make sure they are correct.
 Nothing else will screw up the convergence of your analysis or optimization more quickly than bad derivatives.
-OpenMDAO provides a way to check your partial derivatives so you can make sure they are right.
+OpenMDAO provides a way to check your partial derivatives so you can make sure they are .
```

Comment:
"OpenMDAO provides an helper function to make it easier to verify your partial derivatives"

Reply:
that's better.  I was obviously thinking of how to reword it, didn't finish the sentence.